src/subdomains/generic/user/models/user-data/user-data.entity.ts

```diff
@@ -237,6 +237,9 @@ export class UserData extends IEntity {
   @Column({ length: 256, nullable: true })
   identDocumentType: string;
 
+  @Column({ length: 256, nullable: true })
+  kycClients: string; // semicolon separated wallet id's
```

Comment:
KYC client names need to be added to the `KycInfoDto`

Reply:
Which dto do you mean exactly? Cannot find a KycInfoDto

Reply:
KycClient names? So we have to load every wallet to get the name?

Reply:
Haha, true, it's `KycLevelDto`.

Reply:
Yes, I would set the names.